spec/renders/curation_concerns/renderers/external_link_attribute_renderer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+
+describe CurationConcerns::Renderers::ExternalLinkAttributeRenderer do
+  let(:field) { :name }
+  let(:renderer) { described_class.new(field, ['http://example.com']) }
+
+  describe "#attribute_to_html" do
+    subject { Nokogiri::HTML(renderer.render) }
+    let(:expected) { Nokogiri::HTML(tr_content) }
+
+    let(:tr_content) {
+      "<tr><th>Name</th>\n" \
+       "<td><ul class='tabular'>" \
+       "<li class=\"attribute name\">"\
+       "<a href=\"http://example.com\">"\
+       "<span class='glyphicon glyphicon-new-window'></span>&nbsp;"\
+       "http://example.com</a></li>\n" \
+       "</ul></td></tr>"
+    }
+    it { expect(renderer).not_to be_microdata(field) }
```

Comment:
Is this test necessary?  There are an infinite number of negative assertions we could make about this code, but few of them would have any value.